docs/api/session.md

```diff
@@ -50,11 +50,11 @@ const session = require('electron').session;
 const ses = session.fromPartition('persist:name');
 ```
 
-### Instance Events
+## Events
```

Comment:
Ditto, want to make sure we're still representing it correctly with these updates (per Chengs comments in #5168)